src/components/ItemDetails.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams } from "react-router-dom";
+import styles from "./ItemDetails.module.css";
+import { useEffect, useState } from "react";
+import { Loader } from "./Loader";
+
+const BASE_URL = "http://localhost:9000";
+
+export const ItemDetails = () => {
+  const { id } = useParams();
+  const [isLoading, setIsLoading] = useState(false);
+  const [chosenItem, setChosenItem] = useState("");
```

Comment:
its pretty good, now you can change '[chosenItem, setChosenItem]' to '[itemData, setItemData]', since we're no longer choosing items, we just have a single item data. Ideally the initial state of this data should be defined as `useState(null)`, then `setIsLoading` won't be needed anymore as you can check `if (itemData === null) return <Loader />;`

Reply:
done